exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
```

Comment:
Always remember to give a brief preamble in the form of a short comment about what the lines of code below are intended to do/demonstrate. It can be tedious but months from now, it will come in handy. Comments assist with projects when people work as a group on code. But good that you are getting the basics right so far !